evap/staff/forms.py

```diff
@@ -1063,6 +1064,21 @@ class Meta:
         }
 
 
+class InfotextForm(forms.ModelForm):
+    class Meta:
+        model = Infotext
+        fields = ("title_de", "title_en", "content_de", "content_en")
+
+    def validate_unique(self):
```

Comment:
As I understand it, the code below doesn't validate the uniqueness, so this seems like the wrong place to put this logic

Reply:
Yes, you are right. The problem is, that I found no other way to annotate fields with an error. The CheckConstraint only creates a non_field_error, so I add them to each empty field.
Another option is to create a custom CheckConstraint (e. g. by inheriting from CheckConstraint) that add the errors to the empty fields automatically, when validation fails. This seems a bit overkill though.
The reason why e. g. FAQ does not need this special logic, is because use simply `blank=false` and let django do its job. We cannot use that, because Infotext can be empty

Reply:
From [the documentation](https://docs.djangoproject.com/en/4.1/ref/forms/validation/):
> The form subclass’s `clean()` method can perform validation that requires access to multiple form fields. This is where you might put in checks such as “if field A is supplied, field B must contain a valid email address”. This method can return a completely different dictionary if it wishes, which will be used as the `cleaned_data`.

And I guess `clean` should work for our purposes here?